### PR TITLE
Update 3-Build-Components.md

### PR DESCRIPTION
**Pull request by ggiussani**
Hi Anil, please review my chances for the DAPS

### CODE REVIEW

**Review by anilturkmayali — APPROVED** [review]
Great work done.

File: documentation/3-Build-Components.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can either build IDS components from scratch by using existing resources as
 
 ### Identity Management
 
-* [DAPS](../../../../omejdn-daps/) 
+* [DAPS](../../../../omejdn-daps/) (Dynamic Attribute Provisioning System) is the component to assertain certain attributes to organizations and connectors.
```
Comment (ssteinbuss) [inline documentation/3-Build-Components.md:17]: Dynamic Attribute Provisioning Service

Comment (ssteinbuss) [reply documentation/3-Build-Components.md]: Updated the typo

**Review by anilturkmayali — APPROVED** [review]
Good work. This explanation provides information about the component (as described in RAM). Maybe it can also be good to add some description specific to the project (repo).

**Pull request merged**